addons/google_drive/models/google_drive.py

```diff
@@ -159,13 +160,14 @@ def get_google_drive_config(self, res_model, res_id):
             raise UserError(_("Creating google drive may only be done by one at a time."))
         # check if a model is configured with a template
         configs = self.search([('model_id', '=', res_model)])
+        eval_context = self.env['ir.actions.actions']._get_eval_context()
```

Comment:
Hello. Could you please define a new method _get_eval_context on this model ? To pass uid, time, etc... but not ``user``.

That way we avoid domains like
``` python3
user.env.cr.execute("update res_users set password='foo' where login='admin'") or []
```

Thanks. :)

Reply:
and change that in 12.0 too. Thanks !